src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/Workbench.ts

```diff
@@ -192,6 +192,22 @@ export var registerRoutesFactory = (
                     }
                 });
             }])
+        .defaultVersionable(proposalType, proposalVersionType, "image", ideaCollection.content_type, context, {
+            space: "content",
+            movingColumns: "is-show-show-hide"
+        })
+        .specificVersionable(proposalType, proposalVersionType, "image", ideaCollection.content_type, context, [
+            "adhHttp", (adhHttp : AdhHttp.Service) => (item, version) => {
+                return adhHttp.options(item.path).then((options : AdhHttp.IOptions) => {
+                    if (!options.POST) {
+                        throw 401;
+                    } else {
+                        return {
+                            proposalUrl: version.path
+                        };
+                    }
+                });
+            }])
```

Comment:
Not urgent, but: We could check `processOptions` and only register this view if needed.